context.go

```diff
@@ -203,7 +210,9 @@ func (c *Context) reset(view byte, ts uint64) {
 
 	n := len(c.Validators)
 	c.ChangeViewPayloads = make([]payload.ConsensusPayload, n)
-	c.CommitPayloads = make([]payload.ConsensusPayload, n)
+	if view == 0 {
```

Comment:
This will break `makeCommit()`.

Reply:
We also may have some problem with `updateExistingPayloads()`, it'll wipe the old (view 0) one out.

Reply:
> This will break makeCommit()

Could you, please, clarify how? I don't see the way it can be broken. If you mean that the lines https://github.com/roman-khimov/dbft/blob/39e3e95e00f5b6a4247dbb48761cda4a92da085a/send.go#L112-L114
could accidentally grab some old commit (from some previous view), then it's impossible because the node is able to commit once and it never changes its view after commit.

> We also may have some problem with updateExistingPayloads(), it'll wipe the old (view 0) one out.

It won't, because it has the following check: https://github.com/roman-khimov/dbft/blob/39e3e95e00f5b6a4247dbb48761cda4a92da085a/dbft.go#L396
And in case if commit from the same view will be removed for some node, it means that the commit is invalid and can't be considered. Am I missing something?